### PR TITLE
Update release json for 7.26.x

### PR DESCRIPTION
**Pull request by ogaca-dd**
### What does this PR do?

Update release json for 7.26.x
Add missing `SECURITY_AGENT_POLICIES_VERSION` in `nightly` section.

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+        "OMNIBUS_SOFTWARE_VERSION": "7.26.x",
+        "OMNIBUS_RUBY_VERSION": "7.26.x",
         "JMXFETCH_VERSION": "0.42.0",
         "JMXFETCH_HASH": "3102eb76aa973e4ac08f9727130979d95eda734876424ca4d92b1c4c43081a1f",
         "MACOS_BUILD_VERSION": "master",
         "WINDOWS_DDNPM_DRIVER": "attestation-signed",
         "WINDOWS_DDNPM_VERSION": "0.98.2.git.41.c46c971",
-        "WINDOWS_DDNPM_SHASUM": "79e641b2078737f6176a50779cb233af6cb2fc86ad6a783c871f1310ff751deb"
+        "WINDOWS_DDNPM_SHASUM": "79e641b2078737f6176a50779cb233af6cb2fc86ad6a783c871f1310ff751deb",
+        "SECURITY_AGENT_POLICIES_VERSION": "master"
```
Comment (olivielpeau) [inline release.json:12]: nit: shouldn't this be pinned to `v0.6` instead? I don't think we want the 7.26.x nightlies to pull all updates to the security policies on `master`

**Review by olivielpeau — APPROVED** [review]
LGTM, just a nit

File: release.json (hunk 1 above)
Comment (ogaca-dd) [reply release.json:12]: Do you think I should also create a branch for MACOS_BUILD_VERSION?

Comment (olivielpeau) [reply release.json:12]: Probably, but I'm not sure how this value is handled in the build, any thoughts @bkabrda?

Comment (albertvaka) [reply release.json:12]: This is the repo: https://github.com/DataDog/datadog-agent-macos-build

You can use the tag `7.26.0-rc.1`.

**Pull request merged**